pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -72,22 +73,16 @@ func findByName(name, namespace string, cfgs []model.Config) *model.Config {
 	return nil
 }
 
-func convertResources(r *KubernetesResources) (IstioResources, error) {
+func convertResources(r *KubernetesResources) IstioResources {
 	result := IstioResources{}
-	gw, routeMap, err := convertGateway(r)
-	if err != nil {
-		return result, fmt.Errorf("failed to convert gateways: %v", err)
```

Comment:
motivation here is that returning an error here blocks the entire push pipeline. We don't want that, we want to propagate the errors into Status on the objects, but that isn't implemented yet